README.md

```diff
@@ -35,8 +35,13 @@ Host puppetmaster.theforeman.org
 Now clone:
 
 ```
-gopass clone --sync gitcli puppetmaster.theforeman.org:/srv/secretsgit/theforeman-release.git theforeman/releases
-gopass clone --sync gitcli puppetmaster.theforeman.org:/srv/secretsgit/theforeman-passwords.git theforeman/shared
+gopass clone puppetmaster.theforeman.org:/srv/secretsgit/theforeman-release.git theforeman/releases
+gopass clone puppetmaster.theforeman.org:/srv/secretsgit/theforeman-passwords.git theforeman/shared
+```
+> :warning: **gopass clone --sync gitcli**: no longer exists on 1.12.z, to sync we can use:
+
+```
+gopass sync
```

Comment:
I'd drop that. `--clone sync` was just to *config* the sync, you still needed `gopass sync` to do the syncing, and now it's set up automatically.